packages/graphql-transformer-common/src/ModelResourceIDs.ts

```diff
@@ -1,9 +1,19 @@
-import { graphqlName, toUpper, toCamelCase, simplifyName } from './util';
+import { graphqlName, toUpper, toCamelCase } from './util';
 import { DEFAULT_SCALARS } from './definition';
 
 export class ModelResourceIDs {
+  static #modelNameMap: Map<string, string>;
```

Comment:
This is fine right now, but I think we should get away from keeping this type of state in static fields.

Reply:
100% agree. This was purely a way to continue using the static ModelTableResourceID method and not have to refactor that all over the place as part of this change